scripts/termux-apk

```diff
@@ -0,0 +1,50 @@
+#!/data/data/com.termux/files/usr/bin/bash
+
+install()
+{
+    apk="$(realpath $@)"
+    if [ ! -f "$apk" ]
+    then
+	echo "$apk does not exist"
+	exit 1
+    fi
+    am start -a android.intent.action.INSTALL_PACKAGE -d "file://$apk"
```

Comment:
I'd propagate exit code of `am` command with `return $?` and put `set -e` at the start of the script. The same about `uninstall` function